[2025-04-13T21:31:57.589+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-13T21:31:57.613+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_dlt_data_pipeline.upload_file scheduled__1987-01-01T00:00:00+00:00 [queued]>
[2025-04-13T21:31:57.624+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_dlt_data_pipeline.upload_file scheduled__1987-01-01T00:00:00+00:00 [queued]>
[2025-04-13T21:31:57.626+0000] {taskinstance.py:2867} INFO - Starting attempt 5 of 6
[2025-04-13T21:31:57.639+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): upload_file> on 1987-01-01 00:00:00+00:00
[2025-04-13T21:31:57.648+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1027) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-13T21:31:57.649+0000] {standard_task_runner.py:72} INFO - Started process 1029 to run task
[2025-04-13T21:31:57.648+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'gcs_dlt_data_pipeline', 'upload_file', 'scheduled__1987-01-01T00:00:00+00:00', '--job-id', '348', '--raw', '--subdir', 'DAGS_FOLDER/first_stage/first_stage.py', '--cfg-path', '/tmp/tmpy7x8d9ae']
[2025-04-13T21:31:57.651+0000] {standard_task_runner.py:105} INFO - Job 348: Subtask upload_file
[2025-04-13T21:31:57.724+0000] {task_command.py:467} INFO - Running <TaskInstance: gcs_dlt_data_pipeline.upload_file scheduled__1987-01-01T00:00:00+00:00 [running]> on host 3330be1269a6
[2025-04-13T21:31:57.877+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gcs_dlt_data_pipeline' AIRFLOW_CTX_TASK_ID='upload_file' AIRFLOW_CTX_EXECUTION_DATE='1987-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='scheduled__1987-01-01T00:00:00+00:00'
[2025-04-13T21:31:57.878+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-13T21:32:26.698+0000] {job.py:229} INFO - Heartbeat recovered after 13.91 seconds
[2025-04-13T21:34:02.288+0000] {logging_mixin.py:190} INFO - Uploaded staging/1987.csv to db_project_data_bucket//tmp/1987_transformed.csv
[2025-04-13T21:34:02.289+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-13T21:34:02.301+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-13T21:34:02.301+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=gcs_dlt_data_pipeline, task_id=upload_file, run_id=scheduled__1987-01-01T00:00:00+00:00, execution_date=19870101T000000, start_date=20250413T213157, end_date=20250413T213402
[2025-04-13T21:34:02.380+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-13T21:34:02.397+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-13T21:34:02.400+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
